app/assets/javascripts/darkswarm/controllers/products_controller.js.coffee

```diff
@@ -45,12 +45,12 @@ Darkswarm.controller "ProductsCtrl", ($scope, $filter, $rootScope, Products, Ord
   $scope.appliedTaxonsList = ->
     $scope.activeTaxons.map( (taxon_id) ->
       Taxons.taxons_by_id[taxon_id].name
-    ).join(" & ") if $scope.activeTaxons?
+    ).join(" #{t('products_or')} ") if $scope.activeTaxons?
```

Comment:
Hey @m-budryte have you tested this in the browser? I'm not sure the `#{...}` syntax will work in coffeescript, and I think we need to concatenate two spaces around the translation, like:
```js
).join(" " + t('products_or') + " ") if $scope.activeTaxons?
```

Reply:
I apologise! It looks like you can use the Ruby interpolation style here. It seems like my `coffeescript` is a bit rusty!